main.py

```diff
@@ -1,12 +1,243 @@
+from flask import json
 import requests
+from requests.api import request
 
 URL = "http://127.0.0.1:5000"
 BACKUP_URL = "https://retro-video-store-api.herokuapp.com"
 
-def main():
-    print("WELCOME TO RETRO VIDEO STORE")
+
+'''
+    =============================================
+    HELPER PRINTS
+    =============================================
+'''
+
+def bar_break():
+    print("\n==========================\n")
+
+def list_options_ee():
+    options = {
+        "1" : "Add Video to Store Stock",
+        "2" : "Edit Video Info",
+        "3" : "Remove Video From Inventory",
+        "4" : "View Current Store Stock",
+        "5" : "View Video Info",
+        "6" : "Add New Customer",
+        "7" : "Edit Existing Customer",
+        "8" : "Delete Existing Customer",
+        "9" : "View Existing Customer Records",
+        "10" : "View All Existing Customers",
+        "11" : "Check Out",
+        "12" : "Check In"
+    }
+
+    bar_break()
+    print("Here are your available options:\n")
+    for choice in options:
+        print(f"Option {choice}. {options[choice]}")
+
+    bar_break()
+
+    return options
+
+def list_options_cust():
+    options = {
+
+    }
     pass
 
+'''
+    =============================================
+    EMPLOYEE OPTION FUNCTIONS
+    =============================================
+'''
+
+def add_video():
+    print("Enter video info below:")
+    request_body = {}
+    request_body["title"] = input("Title: ")
+    request_body["release_date"] = input("Release date: ")
+    request_body["total_inventory"] = input("Total inventory: ")
+
+    response = requests.post(URL +"/videos", json=request_body)
+    print(json.dumps(response.json(), indent=1))
+    return
+
+def edit_video():
+    print("Enter updated video info below:")
+    request_body = {}
+    video_id = input("Video ID: ")
+    request_body["title"] = input("Title: ")
+    request_body["release_date"] = input("Release date: ")
+    request_body["total_inventory"] = input("Total inventory: ")
+
+    response = requests.put(URL +"/videos/" +video_id, json=request_body)
+    print(json.dumps(response.json(), indent=1))
+    return
+
+def remove_video():
+    print("DELETE VIDEO - THIS ACTION CANNOT BE UNDONE")
+    if input("Are you sure? Y/N ") != "Y":
+        print("ACTION CANCELLED")
+        return
+    
+    video_id = input("Video ID: ")
+    response = requests.delete(URL +"/videos/" +video_id)
+    print(json.dumps(response.json(), indent=1))
+    return
+
+def view_video_stock():
+    print("All Videos in Store Stock:")
+    response = requests.get(URL +"/videos")
+    print(json.dumps(response.json(), indent=2))
+    return
+
+def view_single_video():
+    print("Video Info Request:")
+    video_id = input("Video ID: ")
+    response = requests.get(URL +"/videos/" +video_id)
+    print(json.dumps(response.json(), indent=1))
+    return
+
+def add_customer():
```

Comment:
The customer functions are very similar to the video functions.  In cases like this I recommend trying to find ways to share functionality between the two sections.  Writing functions that contain the base functionality that can be called with perhaps lists of the input params and prompts and the url are one way to do this.